src/components/SelectField/SelectField.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { Header, Dropdown, DropdownProps } from 'semantic-ui-react'
+import './SelectField.css'
+
+export type SelectFieldProps = DropdownProps & {
+  label?: string
+}
+
+export class SelectField extends React.PureComponent<SelectFieldProps> {
+  hasAction() {
+    let { loading, error, action, onAction } = this.props
+    return !this.isAddress() && !loading && !error && action && onAction
```

Comment:
`action` and `onAction` are not being used in this component, I think this comes from Field?
It's a bit weird that the linter does not complain